### DISCUSSION TITLE
Problemi di comunicazione ZigBee mediante zigbee2mqtt

### DISCUSSION THREAD

**Discussion by hdmvide** [indomus/forum#3009] [ZigBee, Z-Wave, Thread, Matter]
Salve,
sto usando un raspberry con HA e zigbee2mqtt (Mosquito MQTT, MQTT ecc. Ho seguito le vostre guide) per gestire i miei device zigbee. Associo e utilizzo (apparentemente) i miei device correttamente. Uso l’antenna a Sonoff Dongle-E con firmware RCP.

Ogni tanto noto che i dispositivi non comunicano con il bridge, o meglio, è come se si perdesse la connessione con il bridge. Infatti se, ad esempio, pigio un pulsante ZigBee capita che non arriva l’input ed effettivamente non ho evidenza della pressione del pulsante sulla pancia, in corrispondenza del dispositivo. Poi dopo un po’ torna tutto normale. Mi è anche capitato di pigiare un pulsante, al quale era associata un’automazione, ma questa è stata eseguita dopo 1 minuto dalla pressione dello stesso. Come se l’input fosse arrivato in ritardo, Questo ritardo capita con tutti i dispositivi zigbee e non con un o in particolare.

Alla luce di questi problemi, ho installato un altro HA su un altro Raspberry, sempre con un antenna Sonoff Dongle-E ma con firmware DCP e usando ZHA. Con quest’ultimo funziona tutto molto bene e senza interruzioni.

Qualcuno di voi ha mai avuto questi problemi con zigbee2mqtt?

Grazie

**Comment by snakuzzo** [comment]
Firmware NCP, non DCP.
Comunque anche altri hanno segnalato lo stesso comportamento. Il consiglio è...se non ti interessa thread, usa il firmware NCP e fine.
Per RCP c'è tempo...siamo solo all"inizio

**Reply by hdmvide** [reply]
Si, scusami errore di battitura.
In effetti ho riflashato il dongle mettendo il firmware NCP e adesso su zigbee2mqtt sembra molto stabile.
Evidentemente quel  firmware MultiPan non è ancora maturo.

Grazie per il supporto.
